pkg/psmdb/mongos.go

```diff
@@ -285,6 +293,14 @@ func volumes(cr *api.PerconaServerMongoDB) []corev1.Volume {
 				EmptyDir: &corev1.EmptyDirVolumeSource{},
 			},
 		},
+		{
+			Name: "users-secret-file",
```

Comment:
I think it should be added only if cr version -> 1.8.0
to avoid mongos restart during operator upgrade

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/599/commits/c0bf393699e70c222aee61244a057c5c7b4507aa